README.md

```diff
@@ -1 +1,209 @@
-# Conference-page
\ No newline at end of file
+# Conference-page
+
+# Conference-page
+
+# CapstoneModuleOne
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [Content Common Global Submit] <a name="about-project"></a>
+
+
+**Capstone**  Is the project that I created for the Content Common Global Submit and thier future conference in july. The website is a copied from the tamplate given to us by **[Microverse]** as a milestone project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML 5 thas it's markup languange used for structuring and presenting content on the WWW. 
+- CSS 3 that is used for structuring and styling web pages, especially flexbox and grid.
+
+
+### <a name="walkthrough" href= "">Project Walkthrough</a> 
+
+**[https://www.loom.com/share/f670ab1646e54732bd9ea659289274c3]**
+
+
+### Key Features <a name="key-features"></a>
+
+- **NavBar** : contains the logo of organization and the hamburguer menu that takes you to diferent pages and sections of the main page.
+- **Presentation** : Were you can see the basic information of the conferense
+- **Page Development** : Where you can se the way that the conference will develop and the profesional talking.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ]  As previous reviewer suggested, you should show the piece of code that you are interested and proud of making in the video presentation you've made